ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -82,7 +83,7 @@ public ExitFromInpatientPage goToExitFromInpatient() {
     }
 
     public TransferToWardServicePage goToTransferToWardServicePage() {
-    	clickOn(EXIT_FROM_INPATIENT);
+    	clickOn(TRANSFER_TO_WARD);
```

Comment:
Refactored the element `EXIT_FROM_INPATIENT` to the right page element that can ensure `goToTrasferWardServicePage()` method loads the right page. Otherwise the current page element loads `ExitFromInPatientPage` which is not supposed to be the page to be loaded by the method.

Reply:
> which is not supposed to be the page to be loaded by the method But rather `TransferToWardServicePage`.

Will test it and will confirm with you asap

Reply:
Thanks @sherrif10 for reviewing and confirming. 👍